exercises/high-scores/.meta/solutions/high_scores.rb

```diff
@@ -15,13 +15,7 @@ def latest
     scores.last
   end
 
-  def personal_top
-    scores.sort.reverse.take(3)
-  end
-
-  def report
-    difference = "#{personal_best - latest} short of" if personal_best != latest
-
-    "Your latest score was #{latest}. That's #{difference} your personal best!".squeeze
+  def personal_top_three
+    scores.max(3)
```

Comment:
I _think_ I'd prefer to have the 'minimal solution for approval', not the 'best'.
(Because it seems that some mentors refer to the example, or at least look at it.)

Reply:
This shouldn't be too relevant 😕 These solutions are not for mentor consumption - and even if a mentor does see this I don't see it as a big issue at all. I'd rather keep things fairly separate - mentor notes are one thing, these solutions are another.

What do you think? Do you feel strongly about this? I can revert the change, but I don't want us to start changing these solutions every time we change the mentor notes - unless that's something we decide to do across all tracks.

Reply:
Not _very_ strongly, but let's say: preferrable. Strangely exactly because I agree with you: that we shouldn't start changing these solutions. Unless it's a bad example/ bad practice / outdated practice.  
AFAIK the current line is none of those,  or is it?

Reply:
It's not. I'll change it back.

Reply:
But really, the only reason I didn't commit `max(3)` the first time around is because I didn't know about it 😅

Reply:
> It's not. I'll change it back.

Actually, `scores.sort.reverse.take(3)` does do a bit of extra work, when compared to `max(3)`. 

Also, my point about not starting to change these solutions was specifically about changing them every time we change the mentor notes. I think we should be free to change these as we please, without the mentor notes having much influence. My main point is that (at least for now) they are different things and should change independently, if needed.

Reply:
Ooooh, this was absolutely not meant to be blocking merging this PR! I'm sorry.